AWS/GoCardlessConfig.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ToucanTech\AWS;
+
+use ToucanTech\AWS\SecretsManager\SecretsRepositoryFactory;
+
+class GoCardlessConfig
+{
+    public static function getGoCardlessConfig() {
+        $secretsRepository = SecretsRepositoryFactory::getSecretsManagerRepository();
+        $gcconfig = $secretsRepository->getSecret('dev/gocardless')->toArray()['SecretString'];

```

Comment:
:warning: A couple of points around this area.

1. Some assumptions made that that there will be an array returned and it will contain a key of 'SecretString'. This should fail gracefully and would recommend wrapping this in a  try-catch statement for better exception management. 

2. Variable naming convention the returned item is a secret not really a config and a better name might be `$secret`.

3. We have 'dev/goCardless' hardcoded which does not allow us to change this dependant on the enviroment. I would recommend storing the value in an .env file so that we can change this environmentally so that QA environment could have a developer version of GoCardless to test.